app/controllers/goldencobra/api/v2/articles_controller.rb

```diff
@@ -211,77 +145,84 @@ def breadcrumb
           end
         end
 
-        protected
 
-        # Creates an article from the given article array.
-        # ---------------------------------------------------------------------------------------
-        def create_article(article_param)
 
-          # Input validation
-          return nil unless article_param
-          return nil unless params[:article]
-          return nil unless current_user
-          return nil unless params[:referee_id]
+        private
 
-          # Create a new article
-          new_article = Goldencobra::Article.new(params[:article])
-          new_article.creator_id = current_user.id
+        def find_article
+          @article = Goldencobra::Article.where(creator_id: current_user.id,
+                                               external_referee_id: params[:referee_id]).first
 
-          if params[:article][:article_type]
-            new_article.article_type = params[:article][:article_type]
-          else
-            new_article.article_type = 'Default Show'
-          end
+          render_error("article not found", 423) and return unless @article
```

Comment:
Does not suit our style guide right now, but I will discuss it internally. Perhaps we will adopt this.